scss/components/_date-grid.scss

```diff
@@ -0,0 +1,80 @@
+// Date Range Calendar Grid Date Picker
+//
+// Filter interaction for filter date fields
+//
+
+.date-range-grid {
+  @include clearfix();
+  font-family: $sans-serif;
+  padding: u(2rem 0);
+  display: none;
+
+  &:not(.invalid) {
+
+    &.pick-min li:hover div {
```

Comment:
So the `.pick-min` and `.pick-max` classes are states applied to the whole grid depending on what's being picked? (Just making sure I understand).